test/e2e/verify/endpoint.yaml

```diff
@@ -0,0 +1,19 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+{{- contains .}}
+- id: {{ b64enc "Your_ApplicationName" }}.1_{{ b64enc "{GET}/hello" }}
+  name: '{GET}/hello'
```

Comment:
I‘m curious about the endpoint is `{GET}/hello`, not `GET:/hello`, we have changed spring endpoint name to `METHOD:URI` since java-agent 8.8.0

Reply:
the java agent injected is `java-agent 8.7.0`, should we change it into `java-agent 8.8.0`?

Reply:
Yes, it is better to use 8.8.0 agent. The format of endpoint name is not changed randomly, this is a big decision.

Reply:
ok, I will change it.